composer.json

```diff
@@ -11,8 +11,8 @@
     }
   ],
   "require": {
-    "php": ">=5.6.4",
-    "laravel/lumen-framework": "5.4.*"
+    "php": ">=7.2.0",
```

Comment:
In a similar vein, do we actually require this high a version of php (the move to laravel requires this?) or could we leave it at 5 to provide more broad support?

Reply:
PHP5 is EOL, and the low PHP7 versions are also near end of active support. Supporting lower versions will potentially also cause more bugs... (and we cannot use the new PHP7 functionality).
For the backend I forced composer to php 7.2, since we are not depending on older versions it's a good idea just to go for the latest version.